chapters/config.Rmd

```diff
@@ -113,7 +113,7 @@ The labels on our *Jane Eyre* word frequency plot are fine for viewing on screen
 but they will need to be bigger
 if we want to include the figure in a slideshow or report.
 
-```{r configuration-jane-eyre-default, echo=FALSE, fig.cap="Word frequency distribution for 'Jane Eyre' with default label sizes."}
+```{r configuration-jane-eyre-default, echo=FALSE, fig.cap="Word frequency distribution for Jane Eyre with default label sizes."}
```

Comment:
I put single quotes around the novel's name because I wasn't able to typeset it in italics - I still haven't figured out a way :-/